docs/config.rst

```diff
@@ -10,11 +10,15 @@ them in order with the latest one taking more priority:
 * ``c:\scrapyd\scrapyd.conf`` (Windows)
 * ``/etc/scrapyd/conf.d/*`` (in alphabetical order, Unix)
 * ``scrapyd.conf``
-* ``~/.scrapyd.conf`` (users home directory)
+* ``~/.scrapyd.conf`` (user's home directory)
 
 The configuration file supports the following options (see default values in
 the :ref:`example <config-example>`).
 
+.. note::
+    The user's home directory is not expanded for the user that may be specified
+    with the ``--uid`` option, but for the user invoking ``scrapyd``.
+    
```

Comment:
The --uid option is for twistd, not scrapyd.
I'm not sure that you are supposed to pass options to twistd.
There may be problems such as #70

Your note makes sense
but it may give the impression
that passing additional arguments to twistd
is going to work as expected.
Have you tried using other twistd arguments?